src/Microsoft.ML/Microsoft.ML.csproj

```diff
@@ -9,6 +9,7 @@
     <!-- <IncludeBuildOutput>false</IncludeBuildOutput> -->
     <NoWarn>$(NoWarn);NU5127;NU5128</NoWarn>
     <PackageDescription>ML.NET is a cross-platform open-source machine learning framework which makes machine learning accessible to .NET developers.</PackageDescription>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```

Comment:
This isn't the correct path to the readme file. Thats why the builds are failing. 

I think prefixing with `../..` will fix the problem.

Reply:
Thanks. I just made the suggested update.

Reply:
It looks like the latest change removed it again. Did it not fix it?